files/eni-max-pods.txt

```diff
@@ -209,7 +218,7 @@ g5.12xlarge 737
 g5.16xlarge 234
 g5.24xlarge 737
 g5.2xlarge 58
-g5.48xlarge 737
+g5.48xlarge 345
```

Comment:
Why did this change?

Reply:
I have no idea, I just ran the VPC CNI script: https://github.com/aws/amazon-vpc-cni-k8s/blob/master/scripts/gen_vpc_ip_limits.go and saw that it pulled a new limit here

Reply:
based on the change in pkg/awsutils/vpc_ip_resource_limit.go in this [PR](https://github.com/aws/amazon-vpc-cni-k8s/pull/2235), the ENILimit for g5.48xlarge changed from 15 to 7, double confirmed by checking `DescribeInstanceTypes` API, see the response of NetworkInfo below
```json
 "NetworkInfo": {
              "NetworkPerformance": "100 Gigabit",
              "MaximumNetworkInterfaces": 7,
              "MaximumNetworkCards": 1,
              "DefaultNetworkCardIndex": 0,
              "NetworkCards": [
                  {
                      "NetworkCardIndex": 0,
                      "NetworkPerformance": "100 Gigabit",
                      "MaximumNetworkInterfaces": 7
                  }
              ],
              "Ipv4AddressesPerInterface": 50,
              "Ipv6AddressesPerInterface": 50,
              "Ipv6Supported": true,
              "EnaSupport": "required",
              "EfaSupported": true,
              "EfaInfo": {
                  "MaximumEfaInterfaces": 1
              },
              "EncryptionInTransitSupported": true,
              "EnaSrdSupported": false
          }
```

Reply:
Ah nice, thank you for checking!

Reply:
Pretty odd that this changed, but the API doesn't lie. 👍